cider/tests/primitives/assert.futil

```diff
@@ -0,0 +1,42 @@
+import "primitives/core.futil";
```

Comment:
The test file should go under `<ROOT>/tests/correctness`: https://github.com/calyxir/calyx/tree/main/tests/correctness

You can create a new folder or add it within the the directory directly.